### PR TITLE
Statsd support

### PR DESCRIPTION
**Pull request by xaviershay**
@drcapulet @syamp

### CODE REVIEW

**Comment by drcapulet** [comment]
LGTM

File: server.go [hunk 1]
```diff
@@ -260,6 +301,11 @@ func loadState() (*grange.State, int) {
 	return &state, warnings
 }
 
+func cleanupStatsd() {
+	stats.Close()
+	stats = &noopStatsd{}
```
Comment (strangemonad) [inline server.go]: why are you resetting stats to the no-op statsd here just to create a new client shortly after?

Comment (xaviershay) [inline server.go]: Hygeine, maybe being too paranoid. I didn't want to have a closed stats client hanging around and maybe accidentally being used. (This method also gets called in a `defer` from main.)

Comment (xaviershay) [inline server.go]: Imagine I'm setting it to `nil` instead, which would be more normal in a cleanup routine I think.

**Pull request merged**